RwsCheck.py

```diff
@@ -496,10 +492,9 @@ def find_robots_txt(self, check_sets):
                                         "'noindex' or 'none' tag in its header"
                                         )
                 except Exception as inst:
-                    if exception_retries not in str(inst):
-                        if exception_timeout not in str(inst):
-                            self.error_list.append(
-                                f"Unexpected error for service site: {service_site}; Received error: {inst}")
+                    if exception_timeout not in str(inst):
```

Comment:
Should we remove the carveout for timeouts as well? I think we probably want to surface that error, too.

Reply:
Done.